diff --git a/hw/vendor/openhwgroup_cv32e40p/rtl/cv32e40p_cs_registers.sv b/hw/vendor/openhwgroup_cv32e40p/rtl/cv32e40p_cs_registers.sv
index 25c777e..a2eac66 100644
--- a/hw/vendor/openhwgroup_cv32e40p/rtl/cv32e40p_cs_registers.sv
+++ b/hw/vendor/openhwgroup_cv32e40p/rtl/cv32e40p_cs_registers.sv
@@ -507,7 +507,7 @@ module cv32e40p_cs_registers
 
         // mimpid, Machine Implementation ID
         CSR_MIMPID: begin
-          csr_rdata_int = (FPU || COREV_PULP || COREV_CLUSTER) ? 32'h1 : 'b0;
+          csr_rdata_int = ((FPU == 1) || (COREV_PULP == 1) || (COREV_CLUSTER == 1)) ? 32'h1 : 'b0;
         end
 
         // unimplemented, read 0 CSRs
