diff --git a/hw/fpga/constraints/pynq-z2/pin_assign.xdc b/hw/fpga/constraints/pynq-z2/pin_assign.xdc
index 9f8f7de..2ac2451 100644
--- a/hw/fpga/constraints/pynq-z2/pin_assign.xdc
+++ b/hw/fpga/constraints/pynq-z2/pin_assign.xdc
@@ -10,8 +10,8 @@ set_property -dict {PACKAGE_PIN L19 IOSTANDARD LVCMOS33} [get_ports rst_i]
 
 # LEDs
 set_property -dict {PACKAGE_PIN M14 IOSTANDARD LVCMOS33} [get_ports rst_led_o]
-set_property -dict {PACKAGE_PIN N16 IOSTANDARD LVCMOS33} [get_ports clk_led_o]
-set_property -dict {PACKAGE_PIN R14 IOSTANDARD LVCMOS33} [get_ports exit_valid_o]
+set_property -dict {PACKAGE_PIN R14 IOSTANDARD LVCMOS33} [get_ports clk_led_o]
+set_property -dict {PACKAGE_PIN N16 IOSTANDARD LVCMOS33} [get_ports exit_valid_o]
 set_property -dict {PACKAGE_PIN P14 IOSTANDARD LVCMOS33} [get_ports exit_value_o]
 set_property CLOCK_DEDICATED_ROUTE FALSE [get_nets rst_led_OBUF]
 set_property CLOCK_DEDICATED_ROUTE FALSE [get_nets clk_out_OBUF]
